sdk/util.js

```diff
@@ -7,8 +7,8 @@ const headers = {
     protocol: AVAILABLE_PROTOCOLS.ETH,
 };
 
-const WS_PROXY_URL = "ws://nft-sales-websocket.dappradar.com";
-
+//const WS_PROXY_URL = "ws://nft-sales-websocket.dappradar.com";
+const WS_PROXY_URL = "wss://rpc-mainnet.matic.quiknode.pro";
```

Comment:
dont change utils

Reply:
Hi, 
We had to change the WS_PROXY_URL as the existing one (ws://nft-sales-websocket.dappradar.com) was not returning valid chain data for Polygon.

Reply:
Hi @Arnaslt 
We had to change the WS_PROXY_URL as the existing one (ws://nft-sales-websocket.dappradar.com) was not returning valid chain data for Polygon.

Reply:
@cryptopunks2 same issue for the naming, polygon > matic should solve this I guess

Reply:
@Arnaslt - have tried using matic but not getting any response/blocks

Reply:
ws://nft-sales-websocket.dappradar.com

Reply:
https://etherscan.io/

Reply:
"wss://rpc-mainnet.matic.quiknode.pro"